apis/risingwave/v1alpha1/risingwave_types.go

```diff
@@ -135,6 +135,8 @@ const (
 // MetaStorage defines spec of meta service.
 type MetaStorage struct {
 	Type MetaStorageType `json:"type"`
+	// +optional
+	EtcdEndpoint string `json:"etcdEndpoint"`
```

Comment:
We need an "omitempty" here so that the deserializer won't complain about the omitted field.
```suggestion
	EtcdEndpoint string `json:"etcdEndpoint,omitempty"`
```